urdf/examples/preserve_fixed_joint_example.urdf

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--URDF-->
+<robot name='preserve_fixed_joint_lumping_example'>
+  <link name='base_link'>
+    <inertial>
+      <mass value='0.25' />
+      <inertia ixx='0.01' ixy='0' ixz='0' iyy='0.01' iyz='0' izz='0.01' />
+    </inertial>
+    <collision>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <sphere radius="2"/>
+      </geometry>
+    </collision>
+  </link>
+  <joint name='j1' type='fixed'>
+    <parent link='base_link'/>
+    <child link='end_effector'/>
+    <origin xyz='0 0 1' rpy='0 0 0'/>
+  </joint>
+  <link name='end_effector'>
+    <inertial>
```

Comment:
`<inertia />` block is missing, which deviates from prior example

Reply:
Good catch. Changed in https://github.com/gazebosim/sdf_tutorials/pull/88/commits/8d1ad1715d8576d47d854e6474d6c2c282efd925. The resulting SDFormat now also has the `end_effector_visual`. Not sure why it was missing before.